### PR TITLE
feat(elements): Add /sign-in export, remove hardcoded paths

### PR DESCRIPTION
**Pull request by BRKalow**
## Description

* Moves sign-in components to a `/sign-in` export.
* Also refactors the machine to not rely on hardcoded sign in paths, instead deriving the base path from the Clerk router.
* Removes unnecessary `@clerk/nextjs` dependency

## Checklist

- [x] `npm test` runs as expected.
- [x] `npm run build` runs as expected.
- [x] (If applicable) [JSDoc comments](https://jsdoc.app/about-getting-started.html) have been added or updated for any package exports
- [x] (If applicable) [Documentation](https://github.com/clerk/clerk-docs) has been updated

## Type of change

- [ ] 🐛 Bug fix
- [ ] 🌟 New feature
- [ ] 🔨 Breaking change
- [x] 📖 Refactoring / dependency upgrade / documentation
- [ ] other:

## Packages affected

- [ ] `@clerk/backend`
- [ ] `@clerk/chrome-extension`
- [ ] `@clerk/clerk-js`
- [ ] `@clerk/clerk-expo`
- [ ] `@clerk/fastify`
- [ ] `gatsby-plugin-clerk`
- [ ] `@clerk/localizations`
- [ ] `@clerk/nextjs`
- [ ] `@clerk/clerk-react`
- [ ] `@clerk/remix`
- [ ] `@clerk/clerk-sdk-node`
- [ ] `@clerk/shared`
- [ ] `@clerk/themes`
- [ ] `@clerk/types`
- [ ] `build/tooling/chore`

### CODE REVIEW

**Comment by BRKalow** [comment]
!snapshot

**Comment by clerk-cookie** [comment]
Hey @BRKalow - the snapshot version command generated the following package versions:
| Package | Version |
| --- | --- |
| @clerk/backend | 1.0.1-snapshot.vc65ad98 |
| @clerk/chrome-extension | 1.0.1-snapshot.vc65ad98 |
| @clerk/clerk-js | 5.0.1-snapshot.vc65ad98 |
| @clerk/elements | 0.0.2-snapshot.vc65ad98 |
| @clerk/clerk-expo | 1.0.1-snapshot.vc65ad98 |
| @clerk/fastify | 1.0.1-snapshot.vc65ad98 |
| gatsby-plugin-clerk | 5.0.1-snapshot.vc65ad98 |
| @clerk/localizations | 2.0.1-snapshot.vc65ad98 |
| @clerk/nextjs | 5.0.1-snapshot.vc65ad98 |
| @clerk/clerk-react | 5.0.1-snapshot.vc65ad98 |
| @clerk/remix | 4.0.1-snapshot.vc65ad98 |
| @clerk/clerk-sdk-node | 5.0.1-snapshot.vc65ad98 |
| @clerk/shared | 2.0.1-snapshot.vc65ad98 |
| @clerk/themes | 2.0.1-snapshot.vc65ad98 |
| @clerk/types | 4.0.1-snapshot.vc65ad98 |

Tip: Use the snippet copy button below to quickly install the required packages.
`@clerk/backend`
```sh
npm i @clerk/backend@1.0.1-snapshot.vc65ad98 --save-exact
```
`@clerk/chrome-extension`
```sh
npm i @clerk/chrome-extension@1.0.1-snapshot.vc65ad98 --save-exact
```
`@clerk/clerk-js`
```sh
npm i @clerk/clerk-js@5.0.1-snapshot.vc65ad98 --save-exact
```
`@clerk/elements`
```sh
npm i @clerk/elements@0.0.2-snapshot.vc65ad98 --save-exact
```
`@clerk/clerk-expo`
```sh
npm i @clerk/clerk-expo@1.0.1-snapshot.vc65ad98 --save-exact
```
`@clerk/fastify`
```sh
npm i @clerk/fastify@1.0.1-snapshot.vc65ad98 --save-exact
```
`gatsby-plugin-clerk`
```sh
npm i gatsby-plugin-clerk@5.0.1-snapshot.vc65ad98 --save-exact
```
`@clerk/localizations`
```sh
npm i @clerk/localizations@2.0.1-snapshot.vc65ad98 --save-exact
```
`@clerk/nextjs`
```sh
npm i @clerk/nextjs@5.0.1-snapshot.vc65ad98 --save-exact
```
`@clerk/clerk-react`
```sh
npm i @clerk/clerk-react@5.0.1-snapshot.vc65ad98 --save-exact
```
`@clerk/remix`
```sh
npm i @clerk/remix@4.0.1-snapshot.vc65ad98 --save-exact
```
`@clerk/clerk-sdk-node`
```sh
npm i @clerk/clerk-sdk-node@5.0.1-snapshot.vc65ad98 --save-exact
```
`@clerk/shared`
```sh
npm i @clerk/shared@2.0.1-snapshot.vc65ad98 --save-exact
```
`@clerk/themes`
```sh
npm i @clerk/themes@2.0.1-snapshot.vc65ad98 --save-exact
```
`@clerk/types`
```sh
npm i @clerk/types@4.0.1-snapshot.vc65ad98 --save-exact
```

**Comment by BRKalow** [comment]
!snapshot

**Comment by clerk-cookie** [comment]
Hey @BRKalow - the snapshot version command generated the following package versions:
| Package | Version |
| --- | --- |
| @clerk/backend | 1.0.1-snapshot.v8b7e4f0 |
| @clerk/chrome-extension | 1.0.1-snapshot.v8b7e4f0 |
| @clerk/clerk-js | 5.0.1-snapshot.v8b7e4f0 |
| @clerk/elements | 0.0.2-snapshot.v8b7e4f0 |
| @clerk/clerk-expo | 1.0.1-snapshot.v8b7e4f0 |
| @clerk/fastify | 1.0.1-snapshot.v8b7e4f0 |
| gatsby-plugin-clerk | 5.0.1-snapshot.v8b7e4f0 |
| @clerk/localizations | 2.0.1-snapshot.v8b7e4f0 |
| @clerk/nextjs | 5.0.1-snapshot.v8b7e4f0 |
| @clerk/clerk-react | 5.0.1-snapshot.v8b7e4f0 |
| @clerk/remix | 4.0.1-snapshot.v8b7e4f0 |
| @clerk/clerk-sdk-node | 5.0.1-snapshot.v8b7e4f0 |
| @clerk/shared | 2.0.1-snapshot.v8b7e4f0 |
| @clerk/themes | 2.0.1-snapshot.v8b7e4f0 |
| @clerk/types | 4.0.1-snapshot.v8b7e4f0 |

Tip: Use the snippet copy button below to quickly install the required packages.
`@clerk/backend`
```sh
npm i @clerk/backend@1.0.1-snapshot.v8b7e4f0 --save-exact
```
`@clerk/chrome-extension`
```sh
npm i @clerk/chrome-extension@1.0.1-snapshot.v8b7e4f0 --save-exact
```
`@clerk/clerk-js`
```sh
npm i @clerk/clerk-js@5.0.1-snapshot.v8b7e4f0 --save-exact
```
`@clerk/elements`
```sh
npm i @clerk/elements@0.0.2-snapshot.v8b7e4f0 --save-exact
```
`@clerk/clerk-expo`
```sh
npm i @clerk/clerk-expo@1.0.1-snapshot.v8b7e4f0 --save-exact
```
`@clerk/fastify`
```sh
npm i @clerk/fastify@1.0.1-snapshot.v8b7e4f0 --save-exact
```
`gatsby-plugin-clerk`
```sh
npm i gatsby-plugin-clerk@5.0.1-snapshot.v8b7e4f0 --save-exact
```
`@clerk/localizations`
```sh
npm i @clerk/localizations@2.0.1-snapshot.v8b7e4f0 --save-exact
```
`@clerk/nextjs`
```sh
npm i @clerk/nextjs@5.0.1-snapshot.v8b7e4f0 --save-exact
```
`@clerk/clerk-react`
```sh
npm i @clerk/clerk-react@5.0.1-snapshot.v8b7e4f0 --save-exact
```
`@clerk/remix`
```sh
npm i @clerk/remix@4.0.1-snapshot.v8b7e4f0 --save-exact
```
`@clerk/clerk-sdk-node`
```sh
npm i @clerk/clerk-sdk-node@5.0.1-snapshot.v8b7e4f0 --save-exact
```
`@clerk/shared`
```sh
npm i @clerk/shared@2.0.1-snapshot.v8b7e4f0 --save-exact
```
`@clerk/themes`
```sh
npm i @clerk/themes@2.0.1-snapshot.v8b7e4f0 --save-exact
```
`@clerk/types`
```sh
npm i @clerk/types@4.0.1-snapshot.v8b7e4f0 --save-exact
```

**Comment by BRKalow** [comment]
!snapshot

File: packages/elements/package.json [hunk 1]
```diff
@@ -1,7 +1,6 @@
 {
   "name": "@clerk/elements",
-  "version": "0.0.2-beta-v5.8",
-  "private": true,
```
Comment (BRKalow) [inline packages/elements/package.json:4]: TODO: we can add `"private": "true"` back until we're ready to publish openly.

**Comment by clerk-cookie** [comment]
Hey @BRKalow - the snapshot version command generated the following package versions:
| Package | Version |
| --- | --- |
| @clerk/backend | 1.0.1-snapshot.vcaceed6 |
| @clerk/chrome-extension | 1.0.1-snapshot.vcaceed6 |
| @clerk/clerk-js | 5.0.1-snapshot.vcaceed6 |
| @clerk/elements | 0.0.2-snapshot.vcaceed6 |
| @clerk/clerk-expo | 1.0.1-snapshot.vcaceed6 |
| @clerk/fastify | 1.0.1-snapshot.vcaceed6 |
| gatsby-plugin-clerk | 5.0.1-snapshot.vcaceed6 |
| @clerk/localizations | 2.0.1-snapshot.vcaceed6 |
| @clerk/nextjs | 5.0.1-snapshot.vcaceed6 |
| @clerk/clerk-react | 5.0.1-snapshot.vcaceed6 |
| @clerk/remix | 4.0.1-snapshot.vcaceed6 |
| @clerk/clerk-sdk-node | 5.0.1-snapshot.vcaceed6 |
| @clerk/shared | 2.0.1-snapshot.vcaceed6 |
| @clerk/themes | 2.0.1-snapshot.vcaceed6 |
| @clerk/types | 4.0.1-snapshot.vcaceed6 |

Tip: Use the snippet copy button below to quickly install the required packages.
`@clerk/backend`
```sh
npm i @clerk/backend@1.0.1-snapshot.vcaceed6 --save-exact
```
`@clerk/chrome-extension`
```sh
npm i @clerk/chrome-extension@1.0.1-snapshot.vcaceed6 --save-exact
```
`@clerk/clerk-js`
```sh
npm i @clerk/clerk-js@5.0.1-snapshot.vcaceed6 --save-exact
```
`@clerk/elements`
```sh
npm i @clerk/elements@0.0.2-snapshot.vcaceed6 --save-exact
```
`@clerk/clerk-expo`
```sh
npm i @clerk/clerk-expo@1.0.1-snapshot.vcaceed6 --save-exact
```
`@clerk/fastify`
```sh
npm i @clerk/fastify@1.0.1-snapshot.vcaceed6 --save-exact
```
`gatsby-plugin-clerk`
```sh
npm i gatsby-plugin-clerk@5.0.1-snapshot.vcaceed6 --save-exact
```
`@clerk/localizations`
```sh
npm i @clerk/localizations@2.0.1-snapshot.vcaceed6 --save-exact
```
`@clerk/nextjs`
```sh
npm i @clerk/nextjs@5.0.1-snapshot.vcaceed6 --save-exact
```
`@clerk/clerk-react`
```sh
npm i @clerk/clerk-react@5.0.1-snapshot.vcaceed6 --save-exact
```
`@clerk/remix`
```sh
npm i @clerk/remix@4.0.1-snapshot.vcaceed6 --save-exact
```
`@clerk/clerk-sdk-node`
```sh
npm i @clerk/clerk-sdk-node@5.0.1-snapshot.vcaceed6 --save-exact
```
`@clerk/shared`
```sh
npm i @clerk/shared@2.0.1-snapshot.vcaceed6 --save-exact
```
`@clerk/themes`
```sh
npm i @clerk/themes@2.0.1-snapshot.vcaceed6 --save-exact
```
`@clerk/types`
```sh
npm i @clerk/types@4.0.1-snapshot.vcaceed6 --save-exact
```

**Comment by BRKalow** [comment]
!snapshot

**Comment by clerk-cookie** [comment]
Hey @BRKalow - the snapshot version command generated the following package versions:
| Package | Version |
| --- | --- |
| @clerk/backend | 1.0.1-snapshot.v7e9c51b |
| @clerk/chrome-extension | 1.0.1-snapshot.v7e9c51b |
| @clerk/clerk-js | 5.0.1-snapshot.v7e9c51b |
| @clerk/elements | 0.0.2-snapshot.v7e9c51b |
| @clerk/clerk-expo | 1.0.1-snapshot.v7e9c51b |
| @clerk/fastify | 1.0.1-snapshot.v7e9c51b |
| gatsby-plugin-clerk | 5.0.1-snapshot.v7e9c51b |
| @clerk/localizations | 2.0.1-snapshot.v7e9c51b |
| @clerk/nextjs | 5.0.1-snapshot.v7e9c51b |
| @clerk/clerk-react | 5.0.1-snapshot.v7e9c51b |
| @clerk/remix | 4.0.1-snapshot.v7e9c51b |
| @clerk/clerk-sdk-node | 5.0.1-snapshot.v7e9c51b |
| @clerk/shared | 2.0.1-snapshot.v7e9c51b |
| @clerk/themes | 2.0.1-snapshot.v7e9c51b |
| @clerk/types | 4.0.1-snapshot.v7e9c51b |

Tip: Use the snippet copy button below to quickly install the required packages.
`@clerk/backend`
```sh
npm i @clerk/backend@1.0.1-snapshot.v7e9c51b --save-exact
```
`@clerk/chrome-extension`
```sh
npm i @clerk/chrome-extension@1.0.1-snapshot.v7e9c51b --save-exact
```
`@clerk/clerk-js`
```sh
npm i @clerk/clerk-js@5.0.1-snapshot.v7e9c51b --save-exact
```
`@clerk/elements`
```sh
npm i @clerk/elements@0.0.2-snapshot.v7e9c51b --save-exact
```
`@clerk/clerk-expo`
```sh
npm i @clerk/clerk-expo@1.0.1-snapshot.v7e9c51b --save-exact
```
`@clerk/fastify`
```sh
npm i @clerk/fastify@1.0.1-snapshot.v7e9c51b --save-exact
```
`gatsby-plugin-clerk`
```sh
npm i gatsby-plugin-clerk@5.0.1-snapshot.v7e9c51b --save-exact
```
`@clerk/localizations`
```sh
npm i @clerk/localizations@2.0.1-snapshot.v7e9c51b --save-exact
```
`@clerk/nextjs`
```sh
npm i @clerk/nextjs@5.0.1-snapshot.v7e9c51b --save-exact
```
`@clerk/clerk-react`
```sh
npm i @clerk/clerk-react@5.0.1-snapshot.v7e9c51b --save-exact
```
`@clerk/remix`
```sh
npm i @clerk/remix@4.0.1-snapshot.v7e9c51b --save-exact
```
`@clerk/clerk-sdk-node`
```sh
npm i @clerk/clerk-sdk-node@5.0.1-snapshot.v7e9c51b --save-exact
```
`@clerk/shared`
```sh
npm i @clerk/shared@2.0.1-snapshot.v7e9c51b --save-exact
```
`@clerk/themes`
```sh
npm i @clerk/themes@2.0.1-snapshot.v7e9c51b --save-exact
```
`@clerk/types`
```sh
npm i @clerk/types@4.0.1-snapshot.v7e9c51b --save-exact
```

**Comment by LekoArts** [comment]
This PR was superseded by https://github.com/clerk/javascript/pull/2725 and https://github.com/clerk/javascript/pull/2676

**Pull request closed without merging**